pages/Frequency/Publishing.md

```diff
@@ -1,14 +1,13 @@
 # Announcement Publishing
 
 On Frequency, [Announcements](../DSNP/Announcements.md) are mapped to Schemas which in turn publish Frequency Messages.
-Frequency Messages are either direct Graph Changes from a particular user, or a Batch Publication with a multitude of possible users.
+Frequency Messages are either mutable state changes from a particular user, or a Batch Publication with a multitude of possible users.
```

Comment:
This implies an implementation cannot be DSNP compliant with if users can post their own announcements one at a time. This seems overspecified. What is the justification for preventing this (as opposed to simply supporting/allowing announcements in batches)?

Reply:
This page is for the Frequency spec, not DSNP. I just tweaked it so it didn't refer to Graph Change Announcements anymore. But this whole part of the spec site needs to be overhauled and updated outside the scope of this DIP.